pom.xml

```diff
@@ -85,12 +85,12 @@
             <dependency>
                 <groupId>com.github.cafapi</groupId>
                 <artifactId>swagger-client-templates</artifactId>
-                <version>1.0.0-28</version>
+                <version>1.1.0-169</version>
             </dependency>
             <dependency>
                 <groupId>com.github.cafapi</groupId>
                 <artifactId>swagger-ui-annotation</artifactId>
-                <version>1.0.0-28</version>
+                <version>1.1.0-169</version>
```

Comment:
Why are you migrating minor version? Dosnt seem to be any changes other than dependency updates

Reply:
Do you mean that this should be a patch release?